orchestra/__init__.py

```diff
@@ -29,7 +29,7 @@ def _main(argv):
         main_parser.print_help()
         exit(1)
 
-    return cmd_parser.handler(args)
+    return cmd_parser.handler(args) or 0
```

Comment:
Are we sure we don't want to `assert` we don't `return None` and enforce handlers to return something?

Reply:
I made this suggested change but to be honest I don't know what's best, it clobbers the command handlers code with `return 0` statements which don't seem very pythonic to me. 
On the other hand it does increase consistency and as some say, explicit is better than implicit, so I'm not sure.

Reply:
> Explicit is better than implicit.
> There should be one-- and preferably only one --obvious way to do it.
-- PEP0020

But up to you.

Reply:
Keeping the change